ui/apps/dev-server-ui/src/app/(dashboard)/runs/page.tsx

```diff
@@ -63,6 +64,7 @@ export default function Page() {
         endTime: endTime,
         status: filteredStatus,
         timeField,
+        celQuery,
```

Comment:
unclear how to update the `GetRunsDocument` to generate the correct params.
it needs to accept the `$query` param to make sure the CEL is passed over to the API.

cc @anafilipadealmeida

Reply:
https://github.com/inngest/inngest/blob/main/ui/apps/dev-server-ui/src/coreapi.ts#L251
This one 😄